lib/factory_trace/tracker.rb

```diff
@@ -8,11 +8,13 @@ def initialize
 
     def track!
       ActiveSupport::Notifications.subscribe('factory_bot.run_factory') do |_name, _start, _finish, _id, payload|
-        name = payload[:name].to_s
+        name = payload[:factory].name.to_s
         traits = payload[:traits].map(&:to_s)
+        alias_names = payload[:factory].names.map(&:to_s) - [name]
```

Comment:
You don't need to track `alias_names` in `Tracker` because you will have this data from `ExtractDefined`. Please remove changes related to this.